packages/next/src/server/async-storage/work-store.ts

```diff
@@ -24,6 +24,8 @@ export type WorkStoreContext = {
 
   requestEndedState?: { ended?: boolean }
   isPrefetchRequest?: boolean
+  isRSCRequest?: boolean
```

Comment:
does this need to be added to a work store? `renderCssResource` receives `AppRenderContext` as a prop so it seems like if we add it to the context object in app-render, it'll have the necessary information without